contracts/ibc-reflect-send/schema/execute_msg.json

```diff
@@ -417,118 +405,6 @@
         }
       }
     },
-    "StakingMsg": {
```

Comment:
Ah, so the `ibc-reflect-send` doesn't import the "staking" feature, so it doesn't support relaying these messages.
I would actually add the "staking" feature to cosmwasm-std in Cargo.toml so these are still supported.

Reply:
okay, will add. The same applies for ibc-reflect, right?